modules/ossm-security-cert-manage.adoc

```diff
@@ -44,96 +45,68 @@ spec:
+[NOTE]
+====
+You must restart the bookinfo pods if they were already deployed. Use the following command to restart the pods: `oc -n <bookinfo-project> delete pods --all`.
+
+If the pods are deployed after the secret changes made above, then you can skip this task.
+====
 +
 [source,terminal]
 ----
-$ oc exec -it $RATINGSPOD -c istio-proxy -- /bin/cat /etc/certs/cert-chain.pem > /tmp/pod-cert-chain.pem
+$ sleep 60
+$ oc -n <bookinfo-project> exec "$(oc -n <bookinfo-project> get pod -l app=productpage -o jsonpath={.items..metadata.name})" -c istio-proxy -- openssl s_client -showcerts -connect details:9080 > bookinfo-proxy-cert.txt
+$ sed -n '/-----BEGIN CERTIFICATE-----/{:start /-----END CERTIFICATE-----/!{N;b start};/.*/p}' bookinfo-proxy-cert.txt > certs.pem
+$ awk 'BEGIN {counter=0;} /BEGIN CERT/{counter++} { print > "proxy-cert-" counter ".pem"}' < certs.pem
 ----
 +
-The file `/tmp/pod-cert-chain.pem` contains the workload certificate and the CA certificate propagated to the pod.
-+
-. Verify the root certificate is the same as the one specified by the Operator. Replace `<path>` with the path to your certificates.
+You should have three files in your working dir: `proxy-cert-1.pem`, `proxy-cert-2.pem` and `proxy-cert-3.pem`.
+
+. Verify that the root certificate is the same as the one specified by the administrator. Replace `<path>` with the path to your certificates.
 +
 [source,terminal]
 ----
 $ openssl x509 -in <path>/root-cert.pem -text -noout > /tmp/root-cert.crt.txt
+$ openssl x509 -in ./proxy-cert-3.pem -text -noout > /tmp/pod-root-cert.crt.txt
+$ diff -s /tmp/root-cert.crt.txt /tmp/pod-root-cert.crt.txt
 ----
+The /tmp/root-cert.crt.txt and /tmp/pod-root-cert.crt.txt flies are identical.
+
+
+. Verify that the CA certificate is the same as the one specified by the administrator. Replace `<path>` with the path to your certificates.
 +
 [source,terminal]
 ----
-$ openssl x509 -in /tmp/pod-root-cert.pem -text -noout > /tmp/pod-root-cert.crt.txt
-----
-+
-[source,terminal]
-----
-$ diff /tmp/root-cert.crt.txt /tmp/pod-root-cert.crt.txt
-----
-+
-Expect the output to be empty.
-+
-. Verify the CA certificate is the same as the one specified by Operator. Replace `<path>` with the path to your certificates.
-+
-[source,terminal]
-----
-$ sed '0,/^-----END CERTIFICATE-----/d' /tmp/pod-cert-chain.pem > /tmp/pod-cert-chain-ca.pem
-----
-+
-[source,terminal]
-----
-$ openssl x509 -in <path>/ca-cert.pem -text -noout > /tmp/ca-cert.crt.txt
-----
-+
-[source,terminal]
-----
-$ openssl x509 -in /tmp/pod-cert-chain-ca.pem -text -noout > /tmp/pod-cert-chain-ca.crt.txt
-----
-+
-[source,terminal]
-----
-$ diff /tmp/ca-cert.crt.txt /tmp/pod-cert-chain-ca.crt.txt
+$ openssl x509 -in ./proxy-cert-2.pem -text -noout > /tmp/pod-cert-chain-ca.crt.txt
+$ diff -s /tmp/ca-cert.crt.txt /tmp/pod-cert-chain-ca.crt.txt
 ----
-+
-Expect the output to be empty.
-+
+The /tmp/ca-cert.crt.txt and /tmp/pod-cert-chain-ca.crt.txt files are identical.
```

Comment:
And this was where I got stuck again.  Something isn't correct here.  I don't know if it's the command or formatting, but this is the point where things didn't work for me.

Reply:
As discussed, I think we're missing a line in the comparison. From #36414

$ openssl x509 -in <path>/ca-cert.pem -text -noout > /tmp/ca-cert.crt.txt
$ openssl x509 -in ./proxy-cert-2.pem -text -noout > /tmp/pod-cert-chain-ca.crt.txt
$ diff -s /tmp/ca-cert.crt.txt /tmp/pod-cert-chain-ca.crt.txt

Result
Files /tmp/ca-cert.crt.txt and /tmp/pod-cert-chain-ca.crt.txt are identical

Reply:
edited these lines